internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -5933,6 +5933,166 @@ func TestHandleTSpendVotes(t *testing.T) {
 	}})
 }
 
+func TestHandleTxFeeInfo(t *testing.T) {
+	t.Parallel()
+
+	txDesc := &mempool.TxDesc{
+		StartingPriority: 0,
+	}
+	txDesc.Tx = dcrutil.NewTx(block432100.Transactions[1])
+	txDesc.Type = stake.TxTypeRegular
+	txDesc.Added = time.Time{}
+	fee, _ := dcrutil.NewAmount(0.0030)
+	txDesc.Fee = int64(fee)
+
+	blocks := uint32(1)
+	afterBestHeight := block432100.Header.Height + uint32(1)
+	start := uint32(10)
+	beforeStart := uint32(5)
+	end := uint32(11)
+	heightRange := []chainhash.Hash{block432100.BlockHash()}
+	feeInfoBlocks := []types.FeeInfoBlock{
+		{
+			Height: 432100,
+			Number: 1,
+			Min:    0.00010088,
+			Max:    0.00010088,
+			Mean:   0.00010088,
+			Median: 0.00010088,
+			StdDev: 0,
+		},
+	}
+	feeInfoRange := types.FeeInfoRange{
+		Number: 1,
+		Min:    0.00010088,
+		Max:    0.00010088,
+		Mean:   0.00010088,
+		Median: 0.00010088,
+		StdDev: 0,
+	}
+
+	feeInfoMempool := types.FeeInfoMempool{
+		Number: 1,
+		Min:    0.00665188,
+		Max:    0.00665188,
+		Mean:   0.00665188,
+		Median: 0.00665188,
+		StdDev: 0,
+	}
+
+	testRPCServerHandler(t, []rpcTest{{
+		name:    "handleTxFeeInfo: unable to fetch ticket fee info for block",
+		handler: handleTxFeeInfo,
+		cmd: &types.TxFeeInfoCmd{
+			Blocks:     &blocks,
+			RangeStart: &start,
+			RangeEnd:   &end,
+		},
+		mockChain: func() *testRPCChain {
+			chain := defaultMockRPCChain()
+			chain.blockByHeightErr =
+				fmt.Errorf("unable to fetch block by height")
```

Comment:
Same nit as in the other PR to use `errors.new` here instead for consistency with the other tests, and since no formatting is required.